client/src/components/customer/GoogleMap.jsx

```diff
@@ -58,6 +73,14 @@ export default class GoogleMap extends Component {
           text={this.state.restaurantName}
           src="http://maps.google.com/mapfiles/kml/paddle/red-stars.png"/>
 
+          <img position="absolute"
+            height="33"
+            width="33"
+            lat={this.state.currentLocation.lat}
+            lng={this.state.currentLocation.lng}
+            text="You are Here"
+            src="http://maps.google.com/mapfiles/kml/paddle/red-stars.png"/>
```

Comment:
Could we use a different icon here? Maybe a person icon(or something similar) to differentiate between the restaurant location and the user location?